src/panels/lovelace/editor/lovelace-cards.ts

```diff
@@ -0,0 +1,125 @@
+import { Card } from "./types";
+
+const coreDocumentationURLBase = "https://www.home-assistant.io/lovelace/";
+
+export const coreCards: Card[] = [
+  {
+    type: "alarm-panel",
+    documentationURL: `${coreDocumentationURLBase}alarm-panel/`,
```

Comment:
We should not set this for every core card, just make sure the URLs match the type (like they all do now)